core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1038,6 +1059,13 @@ class SparkContext(config: SparkConf) extends Logging {
    * operation will create many references to the same object.
    * If you plan to directly cache, sort, or aggregate Hadoop writable objects, you should first
    * copy them using a `map` function.
+   * @param path directory to the input data files, the path can be comma separated paths
+   * as a list of inputs
+   * @param minPartitions suggested minimum number of partitions for the resulting RDD
+   * @param km implicitly found Class of the key associated with the `fm` implicit parameter
```

Comment:
The thing is that callers won't be specifying these. They're just an implementation detail related to various generic types. If anything, it's the generic types that should be documented, though they're kind of obvious.

These things aren't Classes, but ClassTags. fm is the parameter; it's not something related to fm. Really, fm is related to F.

If this is all we can say about these types I'd remove them, because I'm not sure anyone who doesn't already understand implicit ClassTags is going to be helped by this.